ocaml/xapi/xapi_host.ml

```diff
@@ -2370,7 +2370,7 @@ let set_iscsi_iqn ~__context ~host ~value =
    * *)
   Db.Host.set_iscsi_iqn ~__context ~self:host ~value ;
   Db.Host.add_to_other_config ~__context ~self:host ~key:"iscsi_iqn" ~value ;
-  Xapi_host_helpers.Configuration.set_initiator_name value
+  ignore (Xapi_host_helpers.Configuration.set_initiator_name value)
```

Comment:
you return an int, but ignore it. I think instead any errors should be handled inside the function (e.g. non-zero exitcode would mean failure). See the point about using forkhelpers though.

Reply:
Thank you for that, Edvin. :)

Reply:
Now this returns a string and error handling appears to be done by the call_script helper function. Are there other changes required here?

Reply:
Now you don't need to ignore here anymore, since the function returns `()` before.

Reply:
You've resolved the conversation, but the `ignore` is still there.

Reply:
Apologies, I thought that I fixed it.